src/apps/explorer/components/common/Search/index.tsx

```diff
@@ -13,14 +13,15 @@ interface SearchProps {
 export const Search: React.FC<React.HTMLAttributes<HTMLDivElement> & SearchProps> = (props) => {
   const { className, searchString = '', submitSearchImmediatly = false } = props
   const [query, setQuery] = useState('')
+  const [showPlaceholder, setShowPlaceholder] = useState(true)
```

Comment:
Sorry if this was discussed somewhere else and I missed it, but do you need this to show the placeholder? 
I would expect to leave the placeholder handling up to HTML. Not the case?
I mean, show the placeholder if it's empty, show the user input if there's any.

Reply:
at first we were need to make the placeholder fully visible on mobiles, and the suggestion was to break the line, as i couldn't do it with a native placeholder i did it with a simple element that behaves like usual placeholder but breaks the line when it needs to.

Reply:
Understood Maria, thanks for the explanation :)